docker/standalone/cpu/Dockerfile

```diff
@@ -33,11 +34,13 @@ RUN git clone -b ${CLONE_TAG} --depth 1 https://github.com/BVLC/caffe.git . && \
     for req in $(cat python/requirements.txt) pydot; do pip install $req; done && \
     mkdir build && cd build && \
     cmake -DCPU_ONLY=1 .. && \
-    make -j"$(nproc)"
+    make all -j"$(nproc)" && \
+    make pycaffe
```

Comment:
CMake builds pycaffe automatically for me when Python is detected. Does it not for you?